docs/content/development/datasketches-aggregators.md

```diff
@@ -48,7 +48,7 @@ druid.extensions.loadList=["druid-datasketches"]
 {
   "type"  : "thetaSketchEstimate",
   "name": <output name>,
-  "fieldName"  : <the name field value of the thetaSketch aggregator>
+  "field"  : <the name field value of the thetaSketch aggregator>
```

Comment:
can you make this text similar to proposed in https://github.com/druid-io/druid/pull/2522 as it is not just a string name but a post agg object in the field?